profiles/schema.py

```diff
@@ -205,13 +206,34 @@ def mutate(self, info, **kwargs):
 
 
 class Query(graphene.ObjectType):
-    profile = graphene.Field(ProfileType)
+    profile = graphene.Field(
+        ProfileType,
+        id=graphene.Argument(graphene.UUID, required=True),
```

Comment:
our `ProfileType` is actually a node, and it will be exposed with graphene's [`GlobalID`](https://github.com/graphql-python/graphene/blob/master/graphene/relay/node.py#L28) as `id`. I think it will be simpler for the frontend to use that global ID for this `profile` query too. you can then use [this class method](https://github.com/graphql-python/graphene/blob/master/graphene/relay/node.py#L111) to resolve it. yeah, it's gonna be more verbose, but probably more consistent.

Reply:
Of course. I was blinded by ID's. Does it make any sense now?